docker-compose.yml

```diff
@@ -133,7 +133,7 @@ services:
     command: /project/build/docker/package.sh
     volumes:
     - ./:/project
-  
+
   package.alpine:
```

Comment:
Do we need to support an arm64 version of Alpine at this time? Do we also need to support ARM32? Perhaps we'll add those later but I want to make sure we think about them when implementing this. The comment in `packages.sh` is one way we can think about building them all the same way

Reply:
Well the purpose in this case is just to add the normal ARM64 architecture, about an alpine future version I don't know. For ARM32 I don't think we want that support, we are not supporting x86 on linux either.